scripts/zones/Kuftal_Tunnel/mobs/Amemet.lua

```diff
@@ -1,52 +1,228 @@
------------------------------------
--- Area: Kuftal Tunnel
---   NM: Amemet
------------------------------------
+---------------------------------
+--  Area: Kuftal Tunnel
+--  NM: Amemet
+---------------------------------
 require("scripts/globals/hunts")
------------------------------------
+---------------------------------
 local entity = {}
 
-local pathNodes =
+local pathNodesA =
 {
-     17.94, -0.03, 137.00,
-      7.12,  0.02, 117.76,
-      4.74,  0.27,  99.64,
-     17.30, -4.22,  54.34,
-     19.45, -8.75,  40.65,
-     25.23, -8.96, 24.95,
-     40.71, -8.75,  21.33,
-     64.58, -0.87,  13.55,
-     88.68, -0.44,  12.39,
-    105.03, -0.01,  26.04,
-    109.88,  0.00,  45.25,
-    104.56,  -1.9,  60.02,
-    101.12, -8.56,  78.06,
-    100.54, -9.48,  99.99,
-     97.68, -8.79, 124.00,
-     93.38, -9.94, 137.06,
-     78.21, -8.57, 140.23,
-     64.04, -3.60, 143.38,
-     49.70, -0.08, 147.42,
-     32.56,  0.02, 150.01,
+    102, -0.191, 3,
+    90.06, -0.28, 12.24,
+    67.60, -0.34, 12.42,
+    54.61, -4.11, 16.89,
+    25.03, -9.28, 23.12,
+    19.43, -7.95, 44.37,
+    18.80, -6.63, 47.77,
+    17.72, -5.14, 51.88,
+    16.93, -3.95, 55.33,
+    16.33, -3.09, 57.46,
+    14.76, -1.78, 60.84,
+    11.86, -0.60, 65.58,
+    -0.52, 0.00, 82.02,
+    3.06, 0.15, 125.56,
+    40.12, 0.000, 158.89,
+    60.35, -4.24, 143.23,
+    79.86, -8.75, 139.89,
+    85.86, -8.82, 138.11,
+    91.87, -8.83, 142.52,
+    101.60, -8.71, 143.17,
+    100.62, -8.75, 78.86,
+    108.29, -4.60, 54.26,
+    118.98, 0.00, 38.12,
+    102.61, 0.92, 1.60,
+    102, -0.191, 3,
+    102, -0.191, 3
 }
 
+local pathNodesB =
+{
+    102, -0.191, 3,
+    109.49, 0.59, -0.07,
+    124.47, 0.00, -42.08,
+    91.93, -5.15, -58.24,
+    66.46, -8.81, -62.91,
+    60.86, -9.02, -73.80,
+    57.09, -9.37, -96.88,
+    38.39, -8.62, -98.74,
+    15.73, -1.02, -94.88,
+    -0.65, 0.00, -84.13,
+    -40.24, 0.00, -38.35,
+    -44.36, 0.68, 13.98,
+    -31.28, -0.23, 48.64,
+    -0.52, 0.00, 82.02,
+    3.06, 0.15, 125.56,
+    40.12, 0.000, 158.89,
+    60.35, -4.24, 143.23,
+    79.86, -8.75, 139.89,
+    85.86, -8.82, 138.11,
+    91.87, -8.83, 142.52,
+    101.60, -8.71, 143.17,
+    100.62, -8.75, 78.86,
+    108.29, -4.60, 54.26,
+    118.98, 0.00, 38.12,
+    102.61, 0.92, 1.60,
+    102, -0.191, 3,
+    102, -0.191, 3
+}
+
+
```

Comment:
Sanity checks don't like double empty lines

Reply:
Understood! Will adjust here in a patch.